src/Money/Formatter.php

```diff
@@ -11,7 +11,7 @@ class Formatter
 {
     public function format($amount)
     {
-        $money = new Money($amount * 100, new Currency(strtoupper($this->getCurrency())));
+        $money = new Money((int) $amount * 100, new Currency(strtoupper($this->getCurrency())));
```

Comment:
```suggestion
        $money = new Money(round($amount * 100), new Currency(strtoupper($this->getCurrency())));
```

Reply:
I totally agree with your solution! It's more precise in that way. After I wrote the merge request I changed it to `round` aswel.